docs/data/data-grid/editing/FullFeaturedCrudGrid.js

```diff
@@ -174,8 +176,8 @@ export default function FullFeaturedCrudGrid() {
             <GridActionsCellItem
               icon={<SaveIcon />}
               label="Save"
-              sx={{
-                color: 'primary.main',
+              style={{
+                color: theme.palette.primary.main,
```

Comment:
The `sx` prop is now gone from the typings, even though it works in practice for some components. To add it back on all the material grid subcomponents, we'd need to wrap them with something like `const MaterialActionCellItem = styled(ActionCellItem)({})`.